version/version.go

```diff
@@ -16,7 +16,7 @@ var (
 	// A pre-release marker for the version. If this is "" (empty string)
 	// then it means that it is a final release. Otherwise, this is a pre-release
 	// such as "dev" (in development), "beta", "rc1", etc.
-	VersionPrerelease = "dev"
+	VersionPrerelease = ""
```

Comment:
umm.. looks like we should automate reinserting `dev`.

Reply:
Maybe just set it at build time?
https://stackoverflow.com/questions/47509272/how-to-set-package-variable-using-ldflags-x-in-golang-build